main/UseCases/InterviewMaker.java

```diff
@@ -120,13 +123,31 @@ public void hireIntern(){
      * @return a String representation of the interviewIntern info.
      */
     public String getInterviewInternInfo(){
-        return currentHRSystem.getInterviewInternNames();
+        StringBuilder res = new StringBuilder();
+
+        ResponseTree<ArrayList<String>> responseTree = this.currentInterviewIntern.getResponseTree();
+        for (ResponseTree<ArrayList<String>> response : responseTree.getChildren()){
+            // if the response tree is a root, display the interns initial response(i.e. name, age & skill info).
+            if (response.isRoot()){
+                res.append(response.getData());
```

Comment:
You are only returning the initial response of the the intern here. You should also return the player's first choice option. see method startOfInterviewPrompt

Reply:
The second bit (if response is not a root), selects the specific part of the response tree that is the question option and returns it